### ISSUE TITLE
this shit doesnt work (java.lang.ClassNotFoundException)

### ISSUE THREAD

**Issue by Hellscaped** [CRLauncher/CRLauncher#1]
[latest.log](https://github.com/CRLauncher/CRLauncher/files/14625725/latest.log)

reproduction steps:
1. run the game from the launcher on linux
2. error

expected behavior:
game runs fine
actual behavior:
errored at launch time

system info:
```
System:
  Host: archbtw Kernel: 6.8.0-arch1-1 arch: x86_64 bits: 64
  Desktop: i3 v: 4.23 Distro: Arch Linux
Machine:
  Type: Convertible System: SAMSUNG product: 940X3M v: P03AGE
    serial: 0RUH91CJ700242
  Mobo: SAMSUNG model: NP940X3M-K01US
    v: SGL9062A07-C01-G001-S0002+10.0.15063 serial: 123490EN400015
    UEFI: American Megatrends v: P03AGE.046.170613.PS date: 06/13/2017
Battery:
  ID-1: BAT1 charge: 0% condition: 49.3/54.4 Wh (90.7%) volts: 6.5 min: 15.4
CPU:
  Info: dual core Intel Core i7-7500U [MT MCP] speed (MHz): avg: 800
    min/max: 400/3500
Graphics:
  Device-1: Intel HD Graphics 620 driver: i915 v: kernel
  Device-2: Silicon Motion 720p HD Camera driver: uvcvideo type: USB
  Display: server: X.Org v: 21.1.11 with: Xwayland v: 23.2.4 driver: X:
    loaded: modesetting dri: iris gpu: i915 resolution: 1: 1920x1080~60Hz
    2: 1920x1080~60Hz
  API: OpenGL v: 4.6 compat-v: 4.5 vendor: intel mesa v: 24.0.2-arch1.2
    renderer: Mesa Intel HD Graphics 620 (KBL GT2)
Network:
  Device-1: Intel Wireless 8265 / 8275 driver: iwlwifi
Drives:
  Local Storage: total: 238.47 GiB used: 174.01 GiB (73.0%)
Info:
  Memory: total: 8 GiB available: 7.67 GiB used: 4.6 GiB (59.9%) igpu: 32 MiB
  Processes: 212 Uptime: 9h 33m Shell: Sudo inxi: 3.3.33
```

**Comment by Hellscaped** [comment]
ignore my fucked up battery i need to re solder the capacitors for it but it works fine on wall power

**Comment by TheEntropyShard** [comment]
Fixed in [Release 0.0.4](https://github.com/CRLauncher/CRLauncher/releases/tag/v0.0.4)
